### ISSUE TITLE
Measure Cost Floor Insulation, Form Incomplete

### ISSUE THREAD

**Issue by fishbaugher** [fishbaugher/WAweb-Issues#52]
The NEAT custom floor insulation data entry is different and there appears to be a bug.  In v8.x we have:

<img width="265" alt="Untitled" src="https://github.com/fishbaugher/WAweb-Issues/assets/479175/333f133f-2656-466e-be57-278b652dd0e2">

and when you go to edit the measure costs for floor insulation, you get the expected three (3) separate cost entries:

<img width="693" alt="Untitled" src="https://github.com/fishbaugher/WAweb-Issues/assets/479175/eb8f4e5f-7047-4c3f-8930-4e7fdc219dcf">

In v10, we have the same custom floor insulation editor, but in this case we have three standard types instead of one, plus one custom type:

<img width="258" alt="Untitled" src="https://github.com/fishbaugher/WAweb-Issues/assets/479175/1e17f81a-f1df-4cce-abec-2a7d69aa56ec">

HOWEVER, the Floor Insulation R11 measure cost detail does not agree with the list of standard and custom insulation types:

<img width="720" alt="Untitled" src="https://github.com/fishbaugher/WAweb-Issues/assets/479175/7de0f98b-d6f7-4420-be9c-3bd290081bb7">

**Comment by fishbaugher** [comment]
There appears to be no way to enter measure costs for the set of defined floor insulation types.

**Comment by eckmanwe** [comment]
Blown insulation (cellulose and fiberglass) are added to support the new "Fill Floor Cavity" measure included for direct alignment with the current Standard Work Specifications. 
As with WAv8.9, to consider R-11/19/etc. insulation other than Fiberglass Batts, simply add the desired insulation to the Floor section of the NEAT Insulation Types. 
No functional change from WAv8.9.

**Comment by fishbaugher** [comment]
Bill,  Thank you for your reply.  I see the blown insulation types (2) in the new fill floor cavity measure cost details.  

<img width="1023" alt="Untitled" src="https://github.com/fishbaugher/WAweb-Issues/assets/479175/ca4d5cfd-735b-4e3d-aa4b-ae0fd179a37f">

This confusion will need to be addressed in training, but in the meantime, it might be helpful to place a note to this effect at the bottom of the custom insulation type form here:

<img width="559" alt="Untitled2" src="https://github.com/fishbaugher/WAweb-Issues/assets/479175/9ade1e02-a8d6-44ab-a3be-dee3059629a1">

**Comment by eckmanwe** [comment]
Agreed, we have noted this for discussion with trainers. At current, we do not have an issue to add a user note to the NEAT Insulation Types window. Please note that we deem this a low priority as it only impacts select users (those who manage/edit libraries - Lead Auditor and Agency Admin), and does not significantly effect daily use. That said, we have added it for discussion with the DOE WA Advisory Group.
